backend/moonstreamapi/actions.py

```diff
@@ -535,3 +542,27 @@ def apply_moonworm_tasks(
             entries=entries_pack,
             timeout=15,
         )
+
+
+def get_query_by_name(query_name: str, token: uuid.UUID) -> str:
+
+    params = {"type": BUGOUT_RESOURCE_QUERY_RESOLVER, "name": query_name}
```

Comment:
Why do we need this variable if this is just hardcoded string?

Reply:
Or put it with `USER_ONBOARDING_STATE` in one place

Reply:
Also I propose to add validation for `query_name` like no space, no special symbols and etc (spaces for sure), bacause this parameter goes to search endpoint of journals.

Reply:
"Also I propose to add validation for query_name like no space, no special symbols and etc (spaces for sure), bacause this parameter goes to search endpoint of journals."
You mean to brood endpoint but yes it's going cross query params. Mainly we already protect user from create some thing like that but not from search that usual it's do in frontend side.

But even if some one try search not existed query with spaces it's our problem?